contracts/Zauction.sol

```diff
@@ -120,28 +147,74 @@ contract ZAuction is Initializable, OwnableUpgradeable {
       bid,
       address(domainRegistrar),
       tokenId,
-      expireBlock
+      expireBlock,
+      domainToken
+    );
+  }
+
+  /// Allows setting of a network token. Network is the same conceptually as a top level domain,
+  /// so a network might be 0://wilder and network token would mean that every subdomain in that network
+  /// e.g. 0://wilder.kitty will use that ERC20 token for bidding and sales.
+  /// @param networkId The top level domainId of a network
+  /// @param newNetworkToken The token to set for purchases of subdomains in that network
+  function setNetworkToken(uint256 networkId, IERC20 newNetworkToken)
```

Comment:
nit:
`domainNetwork` or some other descriptor for this concept might be preferable so we aren't overloading the word network in blockchain world.